# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sparc/silo/ls-c-include-order.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./second/ls.c.orig	2004-06-28 21:28:22.725307784 +0200
+++ ./second/ls.c	2004-06-28 21:28:29.333303216 +0200
@@ -19,10 +19,10 @@
    USA.  */
 
 #include <silo.h>
+#include <linux/types.h>
 #include <stringops.h>
 
 typedef int FILE;
-#include <linux/types.h>
 #include <ext2fs/ext2_fs.h>
 #include <ext2fs/ext2fs.h>
 
